test/boost/cdc_test.cc

```diff
@@ -1064,28 +1129,28 @@ SEASTAR_THREAD_TEST_CASE(test_change_splitting) {
             return res;
         };
 
-        // TODO: add a static column when static row handling is fixed
-
-        cquery_nofail(e, "create table ks.t (pk int, ck int, v1 int, v2 int, m map<int, int>, primary key (pk, ck)) with cdc = {'enabled':true}");
+        cquery_nofail(e, "create table ks.t (pk int, ck int, s int static, v1 int, v2 int, m map<int, int>, primary key (pk, ck)) with cdc = {'enabled':true}");
 
         auto now = api::new_timestamp();
 
         cquery_nofail(e, format(
             "begin unlogged batch"
+            " update ks.t using timestamp {} set s = -1 where pk = 0;"
```

Comment:
Could you add a case when both static and regular column is updated, please?

Reply:
This should already be covered here, because all operations inside the batch are squashed into one mutation. The resulting mutation should update both static and clustering rows.

Reply:
I see. Thanks.